### PR TITLE
Add documentation for BluemixCloudFoundry deployment type

### PR DESCRIPTION
**Pull request by adamkingit**
Signed-off-by: Adam King <rak@us.ibm.com>

### CODE REVIEW

File: user/deployment/bluemixcloudfoundry.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Bluemix CloudFoundry Deployment
+layout: en
+permalink: /user/deployment/bluemixcloudfoundry/
+---
+
+You now have the amazing ability to deploy directly to [IBM Bluemic](https://console.ng.bluemix.net/) after a successful build on Travis CI.
```
Comment (plaindocs) [inline user/deployment/bluemixcloudfoundry.md]: amazing ability?

File: user/deployment/bluemixcloudfoundry.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Getting on the Edge
+
+Proper Bluemix CloudFoundry support is currently included only in the edge version of Travis.  See how to enable it via the `.travis.yml` below.
+
+## The Easy Way
+
+Go grab [the Travis gem from GitHub](https://github.com/travis-ci/travis.rb) and run this command:
+
```
Comment (plaindocs) [inline user/deployment/bluemixcloudfoundry.md]:
```sh
    travis setup bluemixcloudfoundry
    ```

Also add `sh` to the code block near the bottom on line 44

makes more sense here I think

File: user/deployment/bluemixcloudfoundry.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+`travis setup bluemixcloudfoundry`
+
+You will be asked to answer a few simple questions about your CloudFoundry setup, and Travis will take care of the rest!
```
Comment (plaindocs) [inline user/deployment/bluemixcloudfoundry.md]: This does not add much info. Maybe avoid using the word 'simple' and list the data that you'll need?

File: user/deployment/bluemixcloudfoundry.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Open up your newly-created `.travis.yml` and add `edge: true` to enable the deploy tool.  See yml below for an example of how to do this.
```
Comment (plaindocs) [inline user/deployment/bluemixcloudfoundry.md]: >   See yml below for an example of how to do this.

also does not add much. 

Should the  `edge: true` be inside the deploy key?

File: user/deployment/bluemixcloudfoundry.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## The Slightly Harder Way
+
+So you want to write your own `.travis.yml`, fine.  Here is the minimum required to get up and running:
+
+```
```
Comment (plaindocs) [inline user/deployment/bluemixcloudfoundry.md]: Syntax highlighting:

    ```yaml

**Comment by adamkingit** [comment]
Thanks for taking time to review. I used parts of the cloudfoundry doc and apparently have the "amazing ability" to not read the text as closely as I should have. I'll update the text and pull request as you suggest.

File: user/deployment/bluemixcloudfoundry.md (hunk 1 above)
Comment (adamkingit) [inline user/deployment/bluemixcloudfoundry.md]: Done

File: user/deployment/bluemixcloudfoundry.md (hunk 2 above)
Comment (adamkingit) [inline user/deployment/bluemixcloudfoundry.md]: Done

File: user/deployment/bluemixcloudfoundry.md (hunk 3 above)
Comment (adamkingit) [inline user/deployment/bluemixcloudfoundry.md]: Rewritten as suggested

File: user/deployment/bluemixcloudfoundry.md (hunk 4 above)
Comment (adamkingit) [inline user/deployment/bluemixcloudfoundry.md]: Updated the key to set this, and eliminated this from the doc

File: user/deployment/bluemixcloudfoundry.md (hunk 5 above)
Comment (adamkingit) [inline user/deployment/bluemixcloudfoundry.md]: done

**Comment by plaindocs** [comment]
Great, thanks.

**Pull request merged**